readme.txt

```diff
@@ -3,7 +3,7 @@ Contributors: 10up, helen, adamsilverstein, jakemgold
 Tags: simple podcasting, podcasting, podcast, apple podcasts, episode, gutenberg, blocks, block
 Requires at least: 4.6
 Tested up to: 5.9
-Requires PHP: 5.3
+Requires PHP: 7.3
```

Comment:
Curious why we're bumping from 5.3 all the way to 7.3? Doesn't seem like we have any new features we're adding that require 7.3 so just wondering on the reason here? I think for sure we should drop support for 5.3 at this point but maybe we just bump to 7.0? We also need to update the README.md file as well, as that mentions 5.3 as the minimum supported version

Reply:
I agree, we should keep 7.0 support. Updated both readme files and added 7.0 to compatibility testing.

The initial bump 7.3 was made together with composer dependency (PHPUnit 9.5 requires at least PHP 7.3). This is dev-only requirement and of course the plugin itself works well with 7.0.